### PR TITLE
remove openapi generator + add post_list_payment_options

### PR DESCRIPTION
**Pull request by steve-gr4vy**
The openapi generator was broken so I removed it, in the same way we did for the Python SDK.. and replaced all functions with a much more simple request model.

### CODE REVIEW

File: lib/Gr4vyConfig.php [hunk 1]
```diff
@@ -199,4 +212,215 @@ private static function getThumbprint($private_key) {
...
+        $response = $this->post("/payment-methods", $payment_method_request);
+        return $response;
+    }
+    public function getPaymentMethod($payment_method_id) {
+        $response = $this->get("/payment-methods/" . $payment_method_id);
+        return $response;
+    }
+    public function listPaymentMethods() {
+        $response = $this->get("/payment-methods");
+        return $response;
```
Comment (cbetta) [inline lib/Gr4vyConfig.php:355]: @steve-gr4vy would we want to add support for query params?

**Comment by cbetta** [comment]
@steve-gr4vy let's make sure to mark this as a major release when merged.

**Comment by cbetta** [comment]
Note: I think we should also upgrade the Magento plugin to use this new version asap so we can keep supporting the Magento SDK going forward.

File: lib/Gr4vyConfig.php (hunk 1 above)
Comment (steve-gr4vy) [reply lib/Gr4vyConfig.php]: done.

**Review by cbetta — APPROVED** [review]

**Pull request merged**